src/popup/router/components/TokenAmount.vue

```diff
@@ -27,17 +34,16 @@ export default {
       default: undefined,
     },
     large: { type: Boolean },
+    noSymbol: { type: Boolean },
   },
   computed: {
     amountRounded() {
       return +this.amount.toFixed(this.amount < 0.01 ? 9 : 2);
     },
     ...mapState({
       amountFiat(state, { convertToCurrency, formatCurrency }) {
-        if (this.symbol !== 'AE') return false;
-        const converted = convertToCurrency(this.amount);
-        if (converted < 0.01 || this.hideFiat) return false;
```

Comment:
I recall @onvisions asked to not show zero fiat conversion when implementing this. Is it synced with him or the design team?

Reply:
That's how it looks in designs

Reply:
Which designs? Maybe is a different use-case and we need to split this component.